src/root/views/EmergencyThreeWEdit/index.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import {
+  _cs,
+  isDefined,
+} from '@togglecorp/fujs';
+import type { History, Location } from 'history';
+import { match as Match } from 'react-router-dom';
+
+import Page from '#components/Page';
+import BreadCrumb from '#components/breadcrumb';
+import Container from '#components/Container';
+import EmergencyThreeWForm from '#components/EmergencyThreeWForm';
+import useAlert from '#hooks/useAlert';
+import LanguageContext from '#root/languageContext';
+
+import { EmergencyProjectResponse } from '#types';
+
+import styles from './styles.module.scss';
+
+interface Props {
+  className?: string;
+  match: Match<{ projectId?: string }>;
+  history: History;
+  location: Location;
+}
+
+function EmergencyThreeWEdit(props: Props) {
+  const {
+    className,
+    location,
+    match: {
+      params: {
+        projectId,
+      },
+    },
+    history,
+  } = props;
+
+  const alert = useAlert();
+  const { strings } = React.useContext(LanguageContext);
+  const crumbs = React.useMemo(() => [
+    {link: location?.pathname, name: strings.breadCrumbEditProject},
+    {link: '/three-w/all/', name: strings.breadCrumbThreeW},
+    {link: '/', name: strings.breadCrumbHome},
+  ], [
+    strings.breadCrumbHome,
+    strings.breadCrumbThreeW,
+    strings.breadCrumbEditProject,
+    location,
+  ]);
+
+  const handleSubmitSuccess = React.useCallback((project: EmergencyProjectResponse) => {
+    alert.show(
+      strings.threeWUpdateMessage,
+      { variant: 'success' },
+    );
+    window.setTimeout(
+      () => history.push(`/emergency/${project.event}/#activities`),
+      250,
+    );
```

Comment:
Why are we waiting for 250ms?

Reply:
So that the page change is not very abrupt